fluent/test/fixtures_structure/expressions_call_args.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This means that the runtime parser wasn't able to parse the fixture. I'd like to see it fixed. I think it's OK to not enforce the indent in the runtime parser and I'm hopeful that a simple replacement of `skipInlineWS` to `skipWS` will suffice.